CMakePresets.json

```
@@ -10,7 +10,8 @@
         "CMAKE_EXPORT_COMPILE_COMMANDS": "ON",
         "SFML_BUILD_EXAMPLES": "ON",
         "SFML_BUILD_TEST_SUITE": "ON",
-        "SFML_WARNINGS_AS_ERRORS": "ON"
+        "SFML_WARNINGS_AS_ERRORS": "ON",
+        "SFML_ENABLE_STDLIB_ASSERTIONS": "ON"
```

```suggestion
        "SFML_ENABLE_STDLIB_ASSERTIONS": "ON",
        "SFML_WARNINGS_AS_ERRORS": "ON"
```
I'd like to keep these in alphabetical order if you don't mind